src/mesh/RadioInterface.h

```diff
@@ -9,8 +9,9 @@
 
 #define MAX_TX_QUEUE 16 // max number of packets which can be waiting for transmission
 
-#define MAX_RHPACKETLEN 256
-#define LORA_HEADER_LENGTH 16
+#define MAX_LORA_PAYLOAD_LEN 255 // max length of 255 per Semtech's datasheets on SX12xx
+#define MESHTASTIC_HEADER_LENGTH 16
```

Comment:
Shall we redefine this to `sizeof(PacketHeader)` or at least change the assert in `RadioInterface::RadioInterface()` to `MESHTASTIC_HEADER_LENGTH`?